modules/ROOT/pages/dw-operators.adoc

```diff
@@ -749,3 +749,44 @@ payload update {
   }
 }
 ----
+
+[[metadata-assignment-operator]]
+== Metadata Assignment Operator
+
+DataWeave supports the metadata assignment operator `<~`, which enables you to set the metadata of any value without the need of using the `as` operator.
+This is useful for cases where you want to set the metadata and keep the current value type. If coercing to a new type is also required then xref:dataweave-types-coercion.adoc[as] needs to be used.
```

Comment:
```suggestion
This operator is useful when you want to set the metadata and retain the current value type. If you are coercing the value to a new type, use xref:dataweave-types-coercion.adoc[as].
```

Reply:
our doc style advises us that "this" identifies something, i.e., "this apple," "this operator," etc.